### PR TITLE
Add default handler for index page.

### PR DESCRIPTION
**Pull request by NullHypothesis**
This patch adds a handler for the enclave's index page, which informs
the visitor that this is an enclave.  If the application URL is given,
the handler also tells the visitor what code the enclave is running and
how it can be verified.

### CODE REVIEW

File: handlers.go
Comment (gpestana) [inline handlers.go:10]: Could we simplify this block of code to:

```
i += fmt.Sprintf("\nIt runs the following code: %s\n Use the following tool to verify the enclave: https://github.com/brave-experiments/verify-enclave", appURL)
```

**Review by gpestana — APPROVED** [review]
Added a couple comments that I believe could improve readability, but feel free to merge this version -- up to you.

File: handlers.go
Comment (NullHypothesis) [reply handlers.go:9]: Yes, will do!

**Comment by NullHypothesis** [comment]
@gpestana I addressed both of your suggestions. Unfortunately, I need another sign-off before being able to merge this :sweat_smile:

**Review by gpestana — APPROVED** [review]
LGTM, thanks for the improvemenets!

**Pull request merged**
